src/services/users/send-password-reset-link/index.js

```diff
@@ -18,4 +19,7 @@ const sendResetLink = async ({ email, url }) => {
 
 }
 
-module.exports = sendResetLink
+module.exports = {
```

Comment:
@itzgeebee i've stretched you on this 😄 , we're almost there, just a bit of clarification

 the `generatePasswordResetLink` function will still be used in this file

we'll only be exporting the `sendPasswordResetLink` function

```
module.exports = sendPasswordResetLink
```

Reply:
so generatePasswordResetLink is only internal to this file, not exported or used anywhere else

Reply:
@jocrah I hope I did it right this time

Reply:
yea @itzgeebee, thanks